google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddAdCustomizer.java

```diff
@@ -141,7 +142,7 @@ private void runExample(GoogleAdsClient googleAdsClient, AddAdCustomizerParams p
           "Please pass exactly two ad group IDs in the adGroupId parameter.");
     }
 
-    String feedName = "Ad Customizer example feed " + System.currentTimeMillis();
+    String feedName = "Ad Customizer example feed " + CodeSampleHelper.getPrintableDatetime();
```

Comment:
I'm inclined to have all these examples do a `static` import of `CodeSampleHelper.getPrintableDateTime()`.

@nwbirnie , any objections?

Reply:
I think it makes sense so made the change. This makes is much easier to read.